src/lib/wallet/GoodWalletClass.js

```diff
@@ -204,7 +204,9 @@ export class GoodWallet {
     const mainnethttpWeb3provider = Config.ethereum[mainnetNetworkId].httpWeb3provider
     this.web3Mainnet = new Web3(mainnethttpWeb3provider)
     const network = this.config.network
-    const networkId = get(ContractsAddress, network + '.networkId', 122)
+
+    // TODO: remove this when development-goerli in deployment.json will have correct networkiId
+    const networkId = get(ContractsAddress, network + '.networkId', network.includes('goerli') ? 5 : 122)
```

Comment:
Need to go over the init step for the wallet where it initializes the contracts.
Contracts that are not be available should not be initialized otherwise init will fail

Reply:
done